.github/workflows/ci.yml

```
@@ -151,7 +152,8 @@ jobs:
         cd build
         cmake -A x64 -DCMAKE_TOOLCHAIN_FILE=${VCPKG_INSTALLATION_ROOT}/scripts/buildsystems/vcpkg.cmake \
               -DCMAKE_PREFIX_PATH=${GITHUB_WORKSPACE}/install \
-              -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} \
+              -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} \ \
```

```suggestion
              -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} \
```

Here, there's one backlash more, probably.